conf/application.conf

```diff
@@ -74,7 +74,8 @@ metrics {
 mongodb {
   uri = "mongodb://localhost:27017/transit-movements"
   retryAttempts = 0
-  timeToLiveInSeconds = 2592000 # 30 days
+  timeToLiveInSeconds = 5184000 # 60 days
+  replaceIndexes = false
```

Comment:
is this required?

Reply:
I did some research and found that is required please make it true  for local as well otherwise we will end up with exception.